test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1

```diff
@@ -59,18 +59,19 @@ Describe "Test-Connection" -tags "CI" {
             $result2 | Should -BeFalse
         }
 
-        It "Ping fake host" {
+        It 'does not emit errors for an unresolvable address when using -Quiet' {
+            Test-Connection -Quiet -TargetName "fakeHost" -Count 1 | Should -BeFalse
+        }
```

Comment:
The test name is seem not right.

Reply:
I'll give it another pass, I missed a couple things here, it seeems.